include/gatekeeper_main.h

```diff
@@ -47,6 +47,8 @@ extern uint64_t cycles_per_sec;
 extern uint64_t cycles_per_ms;
 extern uint64_t picosec_per_cycle;
 
+extern struct log_ratelimit_state log_ratelimit_states[RTE_MAX_LCORE];
```

Comment:
This prototype is suggesting that `log_ratelimit_states` should go to the ratelimit library. In fact, why don't we have a library for log ratelimit since it has no relationship with the current ratelimit library?

Reply:
Good point!